qa/rpc-tests/test_framework/util.py

```diff
@@ -200,27 +220,31 @@ def log_filename(dirname, n_node, logname):
     return os.path.join(dirname, "node"+str(n_node), "regtest", logname)
 
 def stop_node(node, i):
+    print "Stopping node"+str(i)
     node.stop()
+    print "Waiting for  node"+str(i) + " to cleanly exit..."
     bitcoind_processes[i].wait()
     del bitcoind_processes[i]
 
+
```

Comment:
Can you run the .py through some code normalizer (aka code "beautifier")? So there is no duplicate empty lines.